### DISCUSSION TITLE
how to get the Model Units

### DISCUSSION THREAD

**Discussion by ChristianSiebje12** [haplokuon/netDxf#462] [Q&A]
First time working with the code/ data structure behind .dxf and I have some questions:

- how to get the model units my loaded .dxf was done? (mm,cm,m,mile,km, etc...)
  : here I can find only DrawingVariables.LUnits but it doesn't seem to match all possible options

- my app is mainly in meters and I have to draw my dxf Entities in meters: are the geometry coordinates (position, length, etc) already in 1:1 scale (e.g. If I draw a line with 12cm with my model set to cm, will the Entity.Line have a length of 0.12m and not 12m?)

Thanks for the help :)

**Comment by haplokuon** [comment]
Is the DrawingVariables.InsUnits what you are looking for? The LUnits controls how the measurement of linear dimension are shown. The best place to find information is the official AutoCad documentation that is available online [here](https://help.autodesk.com/view/ACD/2023/ENU/).
